src/Microcosm.js

```diff
@@ -73,20 +74,17 @@ class Microcosm extends Foliage {
     return this
   }
 
-  prepare(action, ...buffer) {
+  prepare(/* action, ...buffer */) {
```

Comment:
Forgive the ignorance, what change/benefit is there to doing it this way in comparison to rest params as you had before?